eng/Versions.props

```diff
@@ -28,7 +28,7 @@
     <VisualStudioEditorPackagesVersion>17.5.42-preview</VisualStudioEditorPackagesVersion>
     <!-- This should generally be set to $(VisualStudioEditorPackagesVersion),
          but sometimes EditorFeatures.Cocoa specifically requires a newer editor build. -->
-    <VisualStudioMacEditorPackagesVersion>17.3.133-preview</VisualStudioMacEditorPackagesVersion>
+    <VisualStudioMacEditorPackagesVersion>17.5.152-preview-g356e9d7141</VisualStudioMacEditorPackagesVersion>
```

Comment:
@genlu @allisonchou what branch should this go into?  main-vs-deps?

Reply:
We should be able to bump this to a standard daily build version, yes (won't have a commit hash suffix)?

Reply:
Yeah, this is a package from a PR build. Let's change this to 17.5.157-preview (or 17.5.170-preview if we want latest).

Reply:
Actually `VisualStudioEditorPackagesVersion` should be bumped and this property's value should be changed to `$(VisualStudioEditorPackagesVersion)`. But that can happen in another PR.

Reply:
Given this (hopefully?) only affects Mac we shouldn't need to worry about vs-deps. If we did bump `VisualStudioEditorPackagesVersion` then we might